examples/identifier_affiliation_mappings_rfc.py

```diff
@@ -0,0 +1,146 @@
+            continue
+        authors = dt.document_authors(document)
+        if authors is None:
+            print(f"No authors in data tracker from {rfc.doc_id}")
+            continue
+        for author in authors:
+            person_uri = str(author.person)
+            # person_email_obj = dt.email(author.email)
+            if author.email is not None:
+                person_email_address = str(dt.email(author.email).address)
+            else:
+                person_email_address = None
+            tmp_ident_list = [person_uri,person_email_address]
+            
+            affiliation_str = None
+            if author.affiliation is None:
+                affiliation_str = "Unknown"
+            else:
+                affiliation_str = author.affiliation
+            if author.affiliation == "":
+                affiliation_str = "Unknown"
+           
+            if affiliation_str not in uniq_affil_raw:
+                uniq_affil_raw.append(affiliation_str)
+            aff_entry = aff.AffiliationEntry(affiliation_str,rfc.date(),None)
+            if aff_entry.name not in uniq_affil_list :
+                uniq_affil_list.append(aff_entry.name)
+                
+            af_mapping = None
+            print(f"Looking for {person_uri} or {person_email_address}")
+            index = None
+            if person_uri in ident_index_map:
+                index=ident_index_map[person_uri]
+            elif person_email_address in ident_index_map:
+                index=ident_index_map[person_email_address]
+            if index is not None:
+                af_mapping = affil_map[index]
+            if af_mapping is not None:
+                print("found")
+                if person_uri not in af_mapping.identifiers:
+                    affil_map[index].add_identifier(person_uri)
+                    ident_index_map[person_uri]=index
+                elif person_email_address not in af_mapping.identifiers and \
+                    person_email_address is not None:
+                    affil_map[index].add_identifier(person_email_address)
+                    ident_index_map[person_email_address] = index
+                affil_map[index].add_affiliation(aff_entry)
+                break # found the mapping 
+            
+            if af_mapping is None:
+                print(f"New mapping: {person_uri}")
+                tmp_ident = [person_uri]
+                if person_email_address is not None:
+                    tmp_ident.append(person_email_address)
+                tmp_afil = [aff_entry]
+                tmp_mapping = aff.AffiliationMap(tmp_ident,tmp_afil)
+                affil_map.append(tmp_mapping)
+                i = len(affil_map)-1
+                for ident in tmp_ident:
+                    ident_index_map[ident] = i
+    
+                
+    print(affil_map[0])            
+    with open("./identifier_affiliation_map_pre_consolidation.json",'w') as f:
```

Comment:
This should be a Python file that's imported, not JSON, else it won't work when used as a package via pip